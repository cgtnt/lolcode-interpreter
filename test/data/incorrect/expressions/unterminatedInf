HAI

VISIBLE ALL OF WIN AN FAL!

KTHXBYE
