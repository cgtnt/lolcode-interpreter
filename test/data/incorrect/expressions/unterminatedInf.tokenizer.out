@1:T_BEGIN:HAI-@3:T_COMMAND_TERMINATOR:
-@3:T_WRITE_STDOUT:VISIBLE-@3:T_BOOL_AND_INF:ALL OF-@3:T_TRUE:WIN-@3:T_AND:AN-@3:T_T_IDENTIFIER:FAL-@3:T_BANG:!-@5:T_COMMAND_TERMINATOR:
-@5:T_END:KTHXBYE-@6:T_COMMAND_TERMINATOR:
-@6:T_EOF:
